pkg/kvstore/etcd_debug.go

```diff
@@ -321,6 +321,16 @@ func etcdDbgOutputIPs(ips []net.IP) string {
 	return buf.String()
 }
 
+type yamlConfig struct {
+	client.Config
+
+	InsecureTransport     bool   `json:"insecure-transport"`
+	InsecureSkipTLSVerify bool   `json:"insecure-skip-tls-verify"`
+	Certfile              string `json:"cert-file"`
+	Keyfile               string `json:"key-file"`
+	TrustedCAfile         string `json:"trusted-ca-file"`
+}
+
 func etcdDbgRetrieveRootCAFile(cfgfile string) (certs [][]byte, err error) {
 	var yc yamlConfig
```

Comment:
Something like:

```suggestion
	var yc struct {
		TrustedCAfile string `json:"trusted-ca-file"`
	}
```

should work as well, and allow to drop the full struct definition.